### PR TITLE
New Role: RequestrrX

### PR DESCRIPTION
**Pull request by MBryde**
Added a RequestrrX role in the same way SonarrX and RadarrX works. To easily deploy multiple instances of requestrr. Heavily lended code from other roles.

### CODE REVIEW

File: roles/requestrrx/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+# Docker Image(s):  darkalfx/requestrr                                  #
+# --                                                                    #
+#         Part of the Cloudbox project: https://cloudbox.works          #
+#########################################################################
+#                   GNU General Public License v3.0                     #
+#########################################################################
+---
+- name: "Gather list of requestrrx Docker containers"
+  shell: "docker ps -a --format '{{ '{{' }} .Names{{ '}}' }}' --filter label=com.github.cloudbox.community=requestrrx | xargs echo -n"
+  register: cloudbox_community_sonarrx
```
Comment (desimaniac) [inline roles/requestrrx/tasks/main.yml:14]: Could give it a new variable name here

File: roles/requestrrx/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  ignore_errors: yes
+
+- name: "Stop all requestrrx Docker containers"
+  shell: "docker stop {{ cloudbox_community_requestrrx.stdout }}"
+  ignore_errors: yes
+  when: (cloudbox_community_requestrrx.stdout | trim | length > 0)
+
+- name: "Remove all requestrrx Docker containers"
+  shell: "docker rm {{ cloudbox_community_requestrrx.stdout }}"
```
Comment (desimaniac) [inline roles/requestrrx/tasks/main.yml:23]: Actually, there is a new name, but it's not being registered above.

File: roles/requestrrx/tasks/main.yml (hunk 1 above)
Comment (MBryde) [reply roles/requestrrx/tasks/main.yml]: Should be fixed now in the newest commit

**Comment by Banjer** [comment]
Seems you are missing an entry in community.yml to provice the correct tag

**Comment by chazlarson** [comment]
Needs tag entry in community.xml.

**Comment by Migz93** [comment]
Hi Bryde,

As Chaz advised, could you please add the role to the community.yml file. I can then test this and merge the PR.

**Comment by Migz93** [comment]
Closing this PR as the changes were made in #238.

**Pull request closed without merging**